tools/abel/validators/README.md

```diff
@@ -0,0 +1,210 @@
+- TRUE if an entity is [reporting its own data](../../../ontology/docs/building_config.md#reporting-physical-devices) and FALSE if [the entity is virtual](../../../ontology/docs/building_config.md#virtual-devices).
+
+`Cloud Device ID` *string*
+- Cloud registry [device numeric identifier](../../../ontology/docs/building_config.md#identifiers). The device numeric ID is automatically created by a Cloud IoT application, and it is globally unique and not editable.
+
+`DBO Namespace` *string* **required**
+- [Namespace](../../../ontology/docs/ontology.md#namespaces) for an entity as defined in the [Digital Buidings Ontology](../../../ontology/README.md). e.g. `HVAC`
+
+`DBO General Type`
+- [General Type](../../../ontology/docs/model.md#general-types) for an entity as defined in the [Digital Buildings Ontology](../../../ontology/README.md). e.g. `AHU`
+
+`DBO Entity Type Name` *string* **required**
+- [Entity Type Name](../../../ontology.md#type-names) for an Entity as defined in the [Digital Buildings Ontology](../../../ontology/README.md). e.g. `VAV_SD_DSP`
+
+### Entity Fields
+
+The `Entity Fields` tab defines fields for reporting and virtual entities defined in the `Entities` tab.
+
+`Entity Code`
+- String
+- **Required**
+- Human-readable code for an entity which a field is a data point of.
+- This code must already be defined in the [Entities](#entities) table.
+- e.g. `AHU-1001`
+
+`Entity Guid`
+- String
+- Optional
+- [UUID4](https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_(random)) value for an entity corresponding to the entity in the Entity Code column.
+
+`Reporting Entity Code`
+- String
+- **Required**
+- Human-readable code for an entity which a field is a data point of.
+- e.g. `AHU-1001`
+
+`Reporting Entity Guid`
+- String
+- Optional
+- [UUID4](https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_(random)) value for an entity correspong to the entity in the Reporting Entity Code Column.
+
+`DBO Standard Field Name`
+- String
+- **Required**
+- Standardized field name belonging to an entity type as defined in the [Digital Buildings Ontology](https://github.com/google/digitalbuildings/tree/master/ontology).
+- e.g. `zone_air_temperature_sensor`
+
+`Reporting Entity Field`
+- String
+- Optional
+- Name of a field as it is reported by a gateway or other reporting entity.
+- e.g. `zone_air_temperature_sensor_1`
+
+`Raw Field Path`
+- String
+- **Required**
+- Path to data point on a reporting entities telemetry payload.
+- e.g. `pointset.points.discharge_fan_run_command.present_value`
+
+`Raw Unit Path`
+- String
+- Optional
+- Path to a data point's units in a telemetry payload
+- e.g. `pointset.points.zone_air_cooling_temperature_setpoint.units`
+
+`DBO Standard Unit Value`
+- String
+- Optional
+- Standardized unit for a standard field as defined in [Digital Buildings Ontology](https://github.com/google/digitalbuildings/tree/master/ontology/yaml/resources/fields).
```

Comment:
you forgot all those :)
ctrl + F: https://github.com/google/digitalbuildings/tree/master/ and then replace accordingly